roxctl/generate/generate.go

```diff
@@ -12,7 +11,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "generate",
 		Short: "(Technology Preview) Commands related to generating different resources.",
```

Comment:
```suggestion
		Short: "Commands related to generating different resources.",
```

I think we might have another Technology Preview we can remove, correct?

Reply:
Thanks! I found two more :) Both will be removed in the other PR. To save time, let me merge this and I will double check in the next PR.